spec/integration/runner/variant-reassignment-example-15.spec.js

```diff
@@ -0,0 +1,98 @@
+import * as utils from '../../utils/helper'
```

Comment:
It looks like this example 15 is pretty much the same as in example 13 - again we are dealing with different staged and current variants so I was wondering if this test is needed?

Reply:
Yes, it is indeed quite similar. Maybe it's not necessary, but we will decide this in the future?
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/19/commits/dca62eb6e3f90b0d0d9348a361cb29ae7ab52f1a

Reply:
yup yup..